### ISSUE TITLE
Cannot build info manual for jinx's README.org

### ISSUE THREAD

**Issue by jian-lin** [melpa/package-build#88]
I use the following recipe to try to build an info manual for jinx:

```elisp
(jinx
 :repo "minad/jinx"
 :files (:defaults "jinx-mod.c" "emacs-module.h" "README.org")
 :org-exports ("README.org")
 :fetcher github)
```

and get an error:

```console
Copying files (->) and directories (=>)
  from /build/working/jinx/
  to /build/jinxExdxvV/jinx-20240724.917
    jinx.el -> jinx.el
    jinx-mod.c -> jinx-mod.c
    emacs-module.h -> emacs-module.h
    README.org -> README.org
Generating /build/working/jinx/README.texi
Command `install-info --dir\=dir /build/jinxExdxvV/jinx-20240724.917/README.org' exited with non-zero exit-code: 1
install-info: No such file or directory for /build/jinxExdxvV/jinx-20240724.917/README.org
```

---

`package-build` [assumes] the exported texinfo file is `README.texi`.  However, it is [actually] `jinx.texi`.

cc @minad

[assumes]: https://github.com/melpa/package-build/blob/d5661f1f1996a893fbcbacb4d290c57acab4fb0e/package-build.el#L1066
[actually]: https://github.com/minad/jinx/blob/45961dfdb2a56573af6e06d023e262530d60c09d/README.org?plain=1#L4

**Comment by tarsius** [comment]
Thanks for the report!

Fixed, you'll have to use `:org-exports (("README.org" "jinx.texi"))`.

Note that while `package-build` now support exporting from org to texi, this won't be used on Melpa.

**Comment by jian-lin** [comment]
> Note that while `package-build` now support exporting from org to texi, this won't be used on Melpa.

Yeah, I see that in the related doc string.

BTW, will MELPA accept a PR adding org info manual support to the jinx recipe?  NixOS uses MELPA recipes and I want to enable building org info manual for MELPA packages in NixOS.

**Comment by tarsius** [comment]
> BTW, will MELPA accept a PR adding org info manual support to the jinx recipe? NixOS uses MELPA recipes and I want to enable building org info manual for MELPA packages in NixOS.

Let's not push for that just yet, please. I am currently wrapping up minor improvements, so that I can afterwards fully concentrate on the big change: proper version strings for snapshots. Once that is done, we can look into this, possibly in one or two months or so.

Making it unnecessary to check in generated texi files was another big ticket I was working on for many years. A month ago I finally pulled the plug and quickly had to back-paddle, the ecosystem just ain't ready. I've decided to land the *code*, but give up on the lobbying and move on. And I am not ready to revisit this so soon, at least not be before the snapshot version string mess has been addressed.

(I am tentatively in favor of adding `:org-exports` to some Melpa's recipes, even though Melpa itself would just ignore them, but I'll have a lot of convincing to do when I get to the version string thing, and don't want to get sidetracked and depleted by this.)

Oh, and I wouldn't want to do that, until I release v5.0.0, which won't happen until the version string changes are ready.

**Comment by jian-lin** [comment]
With more context, I will not push this manual thing then.

Proper snapshot version sounds really great!  Thanks for your work.

**Comment by jian-lin** [comment]
Hi @tarsius, since the snapshot version issue has been fixed and package-build 5.0.0 has been released 🎉, what is the plan on the manual thing?  Maybe we can open an issue to track it somewhere (in the MELPA repo?).

**Comment by tarsius** [comment]
I've added a comment at https://github.com/melpa/melpa/issues/2251#issuecomment-4939195221.
